tobler/tests/test_interpolators.py

```diff
@@ -8,26 +8,35 @@
 from tobler.dasymetric import glm, glm_pixel_adjusted, masked_area_interpolate
 from tobler.area_weighted import area_interpolate
 
-p = quilt3.Package.browse("rasters/nlcd", "s3://quilt-cgs")
-p["nlcd_2011.tif"].fetch()
 
 local_raster = os.getcwd() + "/nlcd_2011.tif"
```

Comment:
```
import os
local_raster = os.path.join(os.getcwd(), "nlcd_2011.tif")    # portability
```

Reply:
tests are passing on travis so i'll make this change and its good